pkg/vscode/v1/Makefile

```
@@ -31,3 +34,6 @@ $(_VSCODE_SETTINGS_FILE): $(VSCODE_SETTINGS_REQ)
 ifneq ($(filter proto,$(MF_LANGUAGES)),)
 	$(MF_ROOT)/pkg/vscode/v1/bin/merge-vscode-proto3-settings "$@" "$(PROTO_PROTOC_PATH)" $^
 endif
+
+$(_VSCODE_SETTINGS_GITIGNORE_FILE):
+	@echo "$(_VSCODE_SETTINGS_FILE)" > $@
```

```suggestion
	@echo "$(basename $(_VSCODE_SETTINGS_FILE))" > $@
```

Since the `.gitignore` and JSON files are in the same directory, we don't want to include the `.vscode/` prefix.